README.md

```diff
@@ -51,250 +60,373 @@ Browsers have proposed a variety of tracking policies and privacy models
 ([Chromium](https://github.com/michaelkleber/privacy-model/blob/master/README.md),
 [Edge](https://blogs.windows.com/msedgedev/2019/06/27/tracking-prevention-microsoft-edge-preview/),
 [Mozilla](https://wiki.mozilla.org/Security/Anti_tracking_policy),
-[WebKit](https://webkit.org/tracking-prevention-policy/)) which scope access to user identity to
-some notion of first-party. In defining this scope, we must balance two goals: the scope should be
-small enough to meet the user's privacy expectations, yet large enough to provide the user's desired
-functionality on the site they are interacting with.
+[WebKit](https://webkit.org/tracking-prevention-policy/)) which scope access to user identity to some notion of domain 
+name displayed in the address bar of the browser (aka first-party). In defining this scope, we must balance two goals: 
+the scope should be small enough to meet the user's privacy expectations, yet large enough to provide the user's desired
+functionality on the site(s) they are interacting with.
+
+One natural scope is the domain name in the top-level domain. However, the website the user is interacting with may be
+deployed across multiple domain names. For example, `https://google.com`, `https://google.co.uk`, and 
+`https://youtube.com` are owned by the same entity, as are `https://apple.com` and `https://icloud.com`, or 
+`https://amazon.com` and `https://amazon.de`, or `https://disney.es`, `https://disney.com`, `https://espn.com`, and 
+`https://hulu.com`, or `https://dove.com` and `https://axe.com`.
+
+We wish to allow user agent data sharing to span multiple domains and origins only when the following conditions are 
+met;
+
+-  where the legal ownership of the registerable domain can be established by the user;
```

Comment:
Above comment also applies to later places in this PR where ownership verification is assumed to be part of the process. In the investigative journalism and forensic accounting fields, verification of ownership is generally considered a hard problem (example:  https://www.transparency.org/en/publications/beneficial-ownership-how-to-find-the-real-owners-of-secret-companies )

Reply:
Indeed. This amendment uses notaries and EV SSL certificates to establish proof. Notaries have not to the best of my knowledge been considered by prior work. As better solutions come forward these can be added to. However we should not constrain innovation by seeking to find the perfect solution to this problem.

Reply:
Can you add material to the UA Policy Proposal page to explain how the IEE would handle this? It's not clear to me how this proof would work.

The current version of UA Policy Proposal includes the TLS Certificate approach under "Alternatives Considered and Discarded." It would be helpful to get a better sense of the capabilities required of the IEE in this version.

Reply:
The UA Policy Proposal document has not yet been updated to align to the revised GDPR Validated Sets README. The latest commit renames these additional documents to add to the opening title in those documents explaining they are yet to be revised. If it would be cleared to delete them temporarily let me know.

There is no IEE in this proposal as an IEE would be a central point of authority, and the proposal likely suffer the same issues as IAB EU / TCF2.